File: CPP/Aksesoris.cpp
#include "Petshop.cpp"

class Aksesoris: public Petshop{
	private:
		string Jenis;
		string Bahan;
		string Warna;
	
	public:
		Aksesoris(){
			
		}
		Aksesoris(string jenis, string bahan, string warna){
			Jenis = jenis;
			Bahan = bahan;
			Warna = warna;
		}
		
		void setJenis(string jenis){Jenis = jenis;}
		void setBahan(string bahan){Bahan = bahan;}
		void setWarna(string warna){Warna = warna;}
		
		string getJenis(){return Jenis;}
		string getBahan(){return Bahan;}
		string getWarna(){return Warna;}
		
		~Aksesoris(){
			
		}
};

File: PHP/Aksesoris.php
<?php
include "Petshop.php";

class Aksesoris extends Petshop{
	private $Jenis;
	private $Bahan;
	private $Warna;
	
	function __construct($jenis = null, $bahan = null, $warna = null){
		if($jenis !== null && $bahan !== null && $warna !== null){
			$this->Jenis = $jenis;
			$this->Bahan = $bahan;
			$this->Warna = $warna;
		}
	}
	
	function setJenis($jenis){$this->Jenis = $jenis;}
	function setBahan($bahan){$this->Bahan = $bahan;}
	function setWarna($warna){$this->Warna = $warna;}
	
	function getJenis(){return $this->Jenis;}
	function getBahan(){return $this->Bahan;}
	function getWarna(){return $this->Warna;}
	
	function __destruct(){
		
	}
}

?>